docs/sdk/helper-sdks/providers/evm.mdx

```diff
@@ -112,6 +112,8 @@ Some of the commonly used L2s and the Ethereum chain ids are listed below.
 | 0x89   | 137     | Polygon Mainnet             |
 | 0xA86A | 43114   | Avalanche C-Chain           |
 | 0xA    | 10      | Optimism                    |
+| 0xE    | 14      | Flare                       |
+| 0x13   | 19      | Flare                       |
```

Comment:
Wouldn't the network be Songbird for 0x13? @jO-Osko

Reply:
It would, fixed.